### PR TITLE
Run4-hgx287 Next step for V16 Geometry of HGCal (EE algorithm)

### PR DESCRIPTION
**Pull request by bsunanda**
#### PR description:

Next step for V16 Geometry of HGCal (EE algorithm)

#### PR validation:

Usee runTheMatrix test workflow 37434.0

#### if this PR is a backport please specify the original PR and why you need to backport that PR:

Nothing special

### CODE REVIEW

**Comment by bsunanda** [comment]
@cmsbuild Please test

**Comment by bsunanda** [comment]
@smuzaffar @silviodonato @qliphy The failure is not due to this PR - it says "external terminal request" after the external file is opened. Please act on this

**Comment by bsunanda** [comment]
@cmsbuild Please test

File: Geometry/HGCalCommonData/plugins/DDHGCalSiliconModule.cc [hunk 1]
```diff
@@ -30,7 +30,7 @@
 #include <unordered_set>
 #include <vector>
 
-//#define EDM_ML_DEBUG
+#define EDM_ML_DEBUG
```
Comment (srimanob) [inline Geometry/HGCalCommonData/plugins/DDHGCalSiliconModule.cc:33]: Do you need this DEBUG to turn-on in the release?

File: Geometry/HGCalCommonData/plugins/dd4hep/DDHGCalSiliconModule.cc [hunk 2]
```diff
@@ -22,7 +22,7 @@
 #include "DetectorDescription/DDCMS/interface/DDutils.h"
 #include "FWCore/MessageLogger/interface/MessageLogger.h"
 
-//#define EDM_ML_DEBUG
+#define EDM_ML_DEBUG
```
Comment (srimanob) [inline Geometry/HGCalCommonData/plugins/dd4hep/DDHGCalSiliconModule.cc:25]: Here too.

**Comment by srimanob** [comment]
Please test

**Comment by bsunanda** [comment]
@cvuosalo @srimanob Can you approve this PR soon?

**Comment by srimanob** [comment]
+Upgrade

**Comment by bsunanda** [comment]
@cvuosalo Please sign

**Comment by cvuosalo** [comment]
+1

**Comment by qliphy** [comment]
+1

**Pull request merged**